src/main/webapp/app/shared/layouts/profiles/profile.service.ts

```diff
@@ -41,14 +41,22 @@ export class ProfileService {
                             profileInfo.inProduction = profileInfo.activeProfiles.includes('prod');
                             profileInfo.openApiEnabled = profileInfo.activeProfiles.includes('openapi');
                         }
+                        if (profileInfo.ribbonEnv === undefined) {
+                            profileInfo.ribbonEnv = '';
+                        }
                         profileInfo.sentry = data.sentry;
                         profileInfo.features = data.features;
                         profileInfo.buildPlanURLTemplate = data.buildPlanURLTemplate;
                         profileInfo.commitHashURLTemplate = data.commitHashURLTemplate;
                         profileInfo.sshCloneURLTemplate = data.sshCloneURLTemplate;
                         profileInfo.sshKeysURL = data.sshKeysURL;
-                        profileInfo.externalUserManagementName = data.externalUserManagementName;
-                        profileInfo.externalUserManagementURL = data.externalUserManagementURL;
+
+                        // Both properties below are mandatory Strings (see profile-info.model.ts)
+                        const externalUserManagementName = data.externalUserManagementName;
+                        profileInfo.externalUserManagementName = externalUserManagementName === undefined ? '' : externalUserManagementName;
```

Comment:
👋 Hey Martin ,

### A) The Nullish coalescing operator ??
Thank you for showing me this operator. It is new to me and seems to be quite handy.
If I am not mistaken I can shorten my code even more:
Instead of e.g. 
`const externalUserManagementName = data.externalUserManagementName;`
`profileInfo.externalUserManagementName = externalUserManagementName === undefined ? '' : externalUserManagementName;`
Simply this one line:
`profileInfo.externalUserManagementName = data.externalUserManagementName ?? '';`
I hope this is correct?

### B) About recreating the problem
![image](https://user-images.githubusercontent.com/65814168/120099421-9dd2c800-c13b-11eb-9ca8-745a31fc72d4.png)
![image](https://user-images.githubusercontent.com/65814168/120099427-a4f9d600-c13b-11eb-868d-b483ebc6bea0.png)
I already talked with @SimonEntholzer about this. It seems to be a very niche problem (just think about the number of times students copy a repository or use the profile.service, this is basically used for every programming exercise and log in, so the numbers could be way bigger. Additionally, I do not touch the file where Sentry catches this issue.
 `(clone-repo-button.component)`
Moreover, I think this PR is big and old enough. If this issue continues to reappear I will look into it once more (in a new PR). 

----
I will look over my description again and adjust it 👍